lib/Script/Plugin.cpp

```diff
@@ -112,6 +112,19 @@ bool Plugin::registerPlugin(void *Handle) {
   return true;
 }
 
+void Plugin::addLinkStat(std::string StatName, std::string Value) {
+  PluginStats[StatName] = Value;
+}
+
+std::string Plugin::getPluginLinkStats() const {
```

Comment:
Plugin class should return the stats in a way such that the consumer can iterate over the stats. And the consumer (in this case, TextLayoutPrinter) should handle the formatting of the plugin stats. We should not mix the responsibilities.